resources/texts/change_log.txt

```diff
@@ -1,3 +1,65 @@
+- Overhanging Wall Speeds, now gives you the ability to tune multiple values. Don’t forget to adjust the Overhanging Wall Angle to start using the setting. 
+- Minimum Layer Time with Overhang and Minimum Overhang Segment Length: Fine-tune the minimum layer time for overhangs.
+- Distance to Walls: Finetune combing movements.
+- Overhanging Wall Speeds: Fine-tune speeds based on different overhang percentages.
+- Pressure Advance Factor Setting: New setting for machine definitions.
+- You can now tune the Bottom Surface Skin, like you can tune the Top Surface Skin! You can now tune Extruder, Layers, Line Width, Pattern, Line Directions, Outer Wall Flow, Inner Wall(s) Flow, Flow, Outer Wall Speed, Inner Wall Speed, Skin Speed, Inner Wall Acceleration, Skin Acceleration, Outer Wall Jerk, Inner Wall Jerk, Skin Jerk, and Monotonic Bottom Surface Order
+- Enable/Disable USB Printing: A preference setting to indicate that you are using the printer over USB functionality. This setting lays the groundwork for automatically disabling USB printing in the next release when it’s not being used.
+
+* Bug fixes:
+- Resolved a crash that occurred when switching materials on Sovol printers.
+- Random Seam no longer favors one side and not is truly random again
+- Reduced the slicing time when no support needs to be generated 
+- Fixed a bug where Seam on Vertex with a User Defined seam position was not working correctly.
+- Gcode replacement with a single line of code no longer produces values in separate lines
+- Setting names that become too long after translation are now truncated.
+- Updated UltiMaker printer logos to align with the current style.
+- The number of decimal places displayed for layer height in the top bar has been reduced.
+- Fixed a bug that caused incorrect retracting and hopping on printers with more than 2 extruders.
+- Improved how fast the settings are loaded in the Settings Visibility window
+- Improved how disallowed areas are taken into account when arranging models on the buildplate
+
+
+* Printer definitions, profiles, and materials:
+- Introduced Visual Intents for the Sketch Sprint
+- Introduced new Extra Fast and Draft profiles for the Sketch Sprint
+- Introduced profiles for Sketch printers for Metallic PLA with improved surface quality (matte vs shiny)
+- Introduce High Speed and High Speed Solid intents for Method, Method X, and Method XL
+- Introduced PC ABS and PC ABS FR materials for Method X and Method XL
+- Introduced Nylon Slide for UltiMaker S Series Printers
+- Updated the Breakaway Build Volume Temperature for UltiMaker Factor 4
+- Introduced Makerbot Replicator +
+- Updated Voron2 printers to include TPU ASA and PVA, contributed by @WCEngineer
+- Enabled Relative extrusion (M82 and M83 commands) for Marlin-flavored GCode, contributed by @EmJay276
+
+Cura 5.10 supports Mac OSX 12 Monterey or higher. This is because the tools that we use to create Cura builds no longer support these versions of Mac OSX.
```

Comment:
The second sentence is confusing, should say "the tools that we use ... no longer support older versions of Mac OSX"